search-app/src/cmr/search/services/query_service.clj

```diff
@@ -275,7 +275,7 @@
                                                       :concept-id concept-id
                                                       :result-format result-format})
           results (qe/execute-query context query)]
-      (when (zero? (:hits results))
+      (when (zero? (get-in results [:hits]))
```

Comment:
How is this different from `(when (zero? (:hits results))`?

Reply:
I don't know. I guess it is a result of multiple commits. At some point it was `(when (zero? (get-in results [:hits :hits])`. It shouldn't have changed.